pkg/build/nodeimage/imageimporter.go

```diff
@@ -47,10 +47,8 @@ func (c *containerdImporter) End() error {
 }
 
 func (c *containerdImporter) Pull(image, platform string) error {
-	// TODO: this should exist with a --no-unpack and some way to operate quietly
-	// without discarding output
 	return c.containerCmder.Command(
-		"ctr", "--namespace=k8s.io", "images", "pull", "--platform="+platform, image,
+		"ctr", "--namespace=k8s.io", "content", "fetch", "--platform="+platform, image,
```

Comment:
This is analogous to a hypothetical `ctr images pull --no-unpack`, fetch and pull have overlap in the API.

Reply:
see: https://github.com/containerd/containerd/pull/8368#issuecomment-1500779446